README.md

```diff
@@ -88,6 +88,48 @@ The easiest way to run release please is as a GitHub action:
   convention, [release-please](https://github.com/googleapis/release-please)
   will start creating Release PRs for you.
 
+#### Automating publication to npm
+
+With a few additions, the Release Please action can be made to publish to
+npm when a Release PR is merged:
+
+```yaml
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bcoe/release-please-action@v1.2.1
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          package-name: test-release-please
+      # The logic below handles the npm publication:
+      - uses: actions/checkout@v2
+        # these if statements ensure that a publication only occurs when
+        # a new release is created:
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org'
+        if: ${{ steps.release.outputs.release_created }}
+      - run: npm ci
+        if: ${{ steps.release.outputs.release_created }}
+      - run: npm publish
```

Comment:
I'll admit to not understanding everything that's happening in this script.  I can say - I feel like this was far more straightforward _without_ a GitHub action with `semantic-release`:
https://github.com/JustinBeckwith/linkinator/blob/master/.github/workflows/ci.yaml#L53

Why is this so much harder?

Reply:
I find that code you linked to be pretty much identical, honestly maybe a little more confusing to me. The publish step is this:

```
      - run: npm publish
        env:
          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
        if: ${{ steps.release.outputs.release_created }}
```

☝️  all it's doing is saying that it should only publish if a release was created in the prior steps.